docs/index.md

```diff
@@ -114,7 +114,29 @@ If the issuer has more than one certificate for the same public key (e.g. partic
 
 Issuers SHALL publish their public keys as JSON Web Key Sets (see [RFC7517](https://tools.ietf.org/html/rfc7517#section-5)), available at `<<iss value from JWS>>` + `/.well-known/jwks.json`, with [Cross-Origin Resource Sharing (CORS)](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin) enabled.
 
-The URL at `<<iss value from JWS>>` SHALL use the `https` scheme and SHALL NOT include a trailing `/`. For example, `https://smarthealth.cards/examples/issuer` is a valid `iss` value (`https://smarthealth.cards/examples/issuer/` is **not**).
+The URL at `<<iss value from JWS>>` SHALL use the `https` scheme, and SHALL use the following TLS version 1.2 or 1.3 configurations:
+* For TLS 1.2, support at least one ciphersuite conforming to the following algorithm choices and _minimum_ cryptographic strength:
+  * Key exchange: ECDHE using a 256-bit curve
+  * Authentication: ECDSA with a 256-bit curve, or RSA with a 2048-bit key
+  * Session cipher: AES with a 128-bit key
+  * Cipher mode: CBC or GCM
+  * Hash algorithm: SHA256
+  * The following recommended ciphersuites adhere to these requirements:
+    * TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
+    * TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
+    * TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
+    * TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
+    * TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256
+    * TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA384
+    * TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256
+    * TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384
+* For TLS 1.3, use one of these ciphersuites:
+    * TLS_AES_256_GCM_SHA384
+    * TLS_AES_128_GCM_SHA256
```

Comment:
```suggestion
    * TLS_AES_128_GCM_SHA256
    * TLS_CHACHA20_POLY1305_SHA256
```
Per [rfc8446](https://datatracker.ietf.org/doc/html/rfc8446#section-9.1), why shouldn't `TLS_CHACHA20_POLY1305_SHA256` be in this list?

Reply:
chacha is not a fips 140-2 approved algorithm so it would likely be a problem for federal entities to require support

Reply:
I kept the proposal to the NIST standardized crypto algs, but we can add the CHACHA20_POLY1305.

Reply:
Superseded by ref to BCP 195.